cmd/get_logs.go

```diff
@@ -362,6 +365,17 @@ func (glc *getLogsCmd) getCloudName() string {
 	return ""
 }
 
+func (glc *getLogsCmd) getControlPanelNodes() []v1.Node {
```

Comment:
Could you please add a test for this method?

Reply:
test added